docs/index.html

```diff
@@ -2,9 +2,9 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>ngx-admin: Free Open Source admin dashboard template based on Angular, Bootstrap</title>
+  <title>The most popular admin dashboard based on Angular 9+ and Nebular.</title>
```

Comment:
Title should be
`Ngx-admin - most popular admin dashboard on Angular 9+ and Nebular.`

Reply:
fix